### PR TITLE
[cmake] Bump Vc to 1.4.3; fixes macOS warnings.

### PR DESCRIPTION
**Pull request by Axel-Naumann**

### CODE REVIEW

File: cmake/modules/SearchInstalledSoftware.cmake [hunk 1]
```diff
@@ -1406,21 +1406,17 @@ if(vc AND NOT Vc_FOUND AND NO_CONNECTION)
...
 
-  if(UNIX)
-    set(VC_PATCH_COMMAND patch -p1 < ${CMAKE_SOURCE_DIR}/cmake/patches/vc-deprecated-and-bit-scan-forward.patch)
-  endif()
-
   ExternalProject_Add(VC
     URL     ${Vc_SRC_URI}
-    URL_HASH SHA256=68e609a735326dc3625e98bd85258e1329fb2a26ce17f32c432723b750a4119f
+    URL_HASH SHA256=988ea0053f3fbf17544ca776a2749c097b3139089408b0286fa4e9e8513e037f
     PATCH_COMMAND ${VC_PATCH_COMMAND}
```
Comment (amadio) [inline cmake/modules/SearchInstalledSoftware.cmake:1420]: You need to drop this command and associated patch, they are in Vc 1.4.2 already, so the patch is failing on 1.4.3.

Comment (Axel-Naumann) [reply cmake/modules/SearchInstalledSoftware.cmake:1420]: Yeah totally did this in my second attempt :-)

Comment (amadio) [reply cmake/modules/SearchInstalledSoftware.cmake:1420]: Yes, not the `${VC_PATCH_COMMAND}` is empty, but you don't need the line
```
PATCH_COMMAND ${VC_PATCH_COMMAND}
```
anymore, that's what I meant. You didn't remove this line in your second attempt.

Comment (Axel-Naumann) [reply cmake/modules/SearchInstalledSoftware.cmake]: Thanks, fixed!

**Review by amadio — APPROVED** [review]
Strange that I had to re-request my own review to be able to approve, but looks good to go.

**Pull request merged**